publiq-ui/pub-checkbox.vue

```diff
@@ -28,11 +28,6 @@
         type: String,
         default: '',
       },
-      label: {
-        type: String,
-        default: '',
-        required: true,
-      },
```

Comment:
There's been some confusion I think. My suggestion was to remove the label altogether, and just let this component be a checkbox. Not to remove the prop but keep the `pub-label`.

Reply:
I know this was a unused prop, I commented on your suggestion